src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const list = [...document.querySelectorAll('li')];
+
+const sortList = (argList) => {
+  argList.sort(
+    (e1, e2) => toNumber(e2.dataset.salary) - toNumber(e1.dataset.salary));
+};
+
+const getEmployees = (argList) => {
+  return argList.map(e => ({
```

Comment:
try to give more understandable variable names

```suggestion
  return argList.map(person => ({
```